From 9f27c52841c673d769c92f72a8a4b02940a79275 Mon Sep 17 00:00:00 2001
From: Chao Zeng <chao.zeng@siemens.com>
Date: Fri, 23 Apr 2021 13:59:51 +0800
Subject: [PATCH 6/6] board: add the defconfig to support flash protect and net
 releated command

enable flash protect and add the net releated command under u-boot shell

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
---
 configs/iot2050_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/iot2050_defconfig b/configs/iot2050_defconfig
index f0ae4bc900..9aade59688 100644
--- a/configs/iot2050_defconfig
+++ b/configs/iot2050_defconfig
@@ -98,6 +98,7 @@ CONFIG_MMC_SDHCI_ADMA=y
 CONFIG_MMC_SDHCI_AM654=y
 CONFIG_DM_SPI_FLASH=y
 CONFIG_SPI_FLASH_SFDP_SUPPORT=y
+CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_SPI_FLASH_WINBOND=y
 # CONFIG_SPI_FLASH_USE_4K_SECTORS is not set
 CONFIG_DM_ETH=y
-- 
2.31.1

